### PR TITLE
fix(islamic-website): add pages metadata, add favicon, lix links

### PR DESCRIPTION
**Pull request by hadzhehsen**
closes: HQW-15, HQW-106

### CODE REVIEW

File: apps/islamic-website/utils/get-news-data.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    filename: null | string;
+  };
+  title: string;
+  main_url: string;
+  content_type: 'PRESS' | 'VIDEO';
+  main_url_text: string;
+}
+
+const STORYBLOK_ACCESS_TOKEN = process.env['STORYBLOK_ACCESS_TOKEN'];
+const VERCEL_ENV = process.env['VERCEL_ENV'];
```
Comment (olegshilov) [inline apps/islamic-website/utils/get-news-data.ts]: lets move this vars to `constants.ts`

File: apps/islamic-website/src/contansts.ts
Comment (olegshilov) [inline apps/islamic-website/src/contansts.ts:1]: move this file to project root

`apps/islamic-website/constants.tsx`

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/islamic-website/utils/get-news-data.ts (hunk 1 above)
Comment (hadzhehsen) [reply apps/islamic-website/utils/get-news-data.ts]: Done

File: apps/islamic-website/src/contansts.ts
Comment (hadzhehsen) [reply apps/islamic-website/src/contansts.ts:1]: Done

**Review by olegshilov — DISMISSED** [review]

**Review by olegshilov — APPROVED** [review]

**Pull request merged**